inst/shiny-apps/abm/server.R

```diff
@@ -62,9 +62,14 @@ server <- function(input, output, session) {
         if (!ABM_IS_PUBLIC)
             orgs <- c(kthid(), orgs)
         
+        # Set selected org to KTH for now. In some future iteration,
+        # we migth want to select for example a school/department
+        # depending on who is logged in
+        selected_diva_org_id <- 177
```

Comment:
could possibly be named `DEFAULT_ORG` or `default_org` to indicate that it is a default rather than user selected setting (I couldn't find any good guidance wrt SNAKECASED_UPPERCASE_CONSTANTS in https://style.tidyverse.org/syntax.html#object-names)